Translations/Internationalization.md

```diff
@@ -0,0 +1,7 @@
+> Internationalization is the process to provide multiple language support to software, in this case WordPress. Internationalization is often abbreviated as i18n, where 18 stands for the number of letters between the first i and the last n.
```

Comment:
```suggestion
> Internationalisation is the process of providing multiple language support to software, in this case WordPress. It's often abbreviated as _i18n_, where 18 stands for the number of letters between the first _i_ and the last _n_.
```

Reply:
Yeah, the "to provide" is really lousy phrasing here, took the whole quote from WP handbook.